parley/src/layout/line/mod.rs

```diff
@@ -90,6 +90,8 @@ pub struct LineMetrics {
     pub descent: f32,
     /// Typographic leading.
     pub leading: f32,
+    /// CSS definition of line height (px value or multiplicative value * font size)
```

Comment:
‘px value’ is not supported, so should be removed from doc comment.

Reply:
Removed